src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -23,6 +23,9 @@ including:
 - Local maximum/minimum Zalesak-type limiting for conservative variables (`local_minmax_variables_cons`)
 - Positivity limiting for conservative variables (`positivity_variables_cons`)
 
+Conservative variables to be limited are passed as a strings, e.g. `local_minmax_variables_cons = ["rho"]`
```

Comment:
```suggestion
Variables to be limited are passed as a vector of strings, e.g. `local_minmax_variables_cons = ["rho"]`
```

Reply:
Thanks. I would prefer to keep the "Conservative" to make sure that this is only for conservative variables. Also, the name of the variable vector contains `cons`.

Reply:
OK. I've updated my suggestion